lib/Test/Nginx/Socket.pm

```diff
@@ -2121,8 +2121,11 @@ sub gen_curl_cmd_from_req ($$) {
         push @args, '-sS';
     }
 
-    if (use_http2($block)) {
+    my $isHttp2 = use_http2($block);
+    if ($isHttp2 && $isHttp2 == 1) {
```

Comment:
We do not use "camel case" identifier names in this Perl project. We use "snake case" consistently.

Reply:
Sorry, I'll change it.